test/unit/test_LIALogicMkTerms.cc

```diff
@@ -21,132 +22,132 @@ class LIALogicMkTermsTest: public ::testing::Test {
 TEST_F(LIALogicMkTermsTest, testIsNumTerm) {
     PTRef c = logic.mkBoolVar("c");
     ASSERT_FALSE(logic.isNumTerm(c));
-    PTRef t = logic.mkNumVar("t");
+    PTRef t = logic.mkIntVar("t");
     ASSERT_TRUE(logic.isNumTerm(t));
-    PTRef e = logic.mkNumTimes(logic.mkNumVar("e"), logic.mkConst(-114));
+    PTRef e = logic.mkTimes(vec<PTRef>{logic.mkIntVar("e"), logic.mkIntConst(-114)});
     ASSERT_TRUE(logic.isNumTerm(e));
     PTRef ite = logic.mkIte(c, t, e);
     ASSERT_TRUE(logic.isNumTerm(ite));
-    PTRef ite_term = logic.mkNumTimes(ite, logic.mkConst(14));
+    PTRef ite_term = logic.mkTimes(vec<PTRef>{ite, logic.mkIntConst(14)});
     ASSERT_TRUE(logic.isNumTerm(ite_term));
 
-    PTRef sum = logic.mkNumPlus(t, e);
+    PTRef sum = logic.mkPlus(vec<PTRef>{t, e});
     ASSERT_FALSE(logic.isNumTerm(sum));
 }
 
 TEST_F(LIALogicMkTermsTest, testDeepLessThan) {
-    PTRef a = logic.mkConst(3);
-    PTRef b = logic.mkConst(-4);
-    PTRef prod2 = logic.mkNumTimes(y, b);
-    PTRef prod1 = logic.mkNumTimes(x, a);
+    PTRef a = logic.mkIntConst(3);
+    PTRef b = logic.mkIntConst(-4);
+    PTRef prod2 = logic.mkTimes(vec<PTRef>{y, b});
+    PTRef prod1 = logic.mkTimes(vec<PTRef>{x, a});
```

Comment:
Also here, all the `vec`s are redundant, no?

Reply:
Fixed.